project.py

```diff
@@ -13,41 +12,67 @@ def main():
             continue
         else:
             break
+    
+    while True:
+        try:
+            rows = int(input("How many rows do you want to have? "))
+            columns = int(input("How many columns( do you want to have? "))
+            continuous_points_for_win = int(input("How many continuous points do you want to be required for a win?"))
+            break
+        except ValueError:
+            pass
+   
+    board = [[" " for _ in range(columns)] for _ in range(rows)]
     while True:
         print_board(board)
-        board = move(board, find_move(board))
-        if not check_win(board):
+        user_move = find_move(board)

```

Comment:
It would be easier to understand here if you did this:
```suggestion
        user_row, user_column = find_move(board)
```